src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -1,11 +1,12 @@
 from collections import defaultdict
 from datetime import datetime, timedelta
 from time import time
-from typing import Mapping
+from typing import List, Mapping, Type
```

Comment:
can we use `list` instead? see https://sentry.slack.com/archives/C05EFP5UE9H/p1697471522627489?thread_ts=1697471336.836359&cid=C05EFP5UE9H